.catppuccin.yaml

```diff
@@ -0,0 +1,27 @@
+# A catppuccin RC for tmux. In beta, specs may change. 
+app_name: tmux
+binary_name: tmux 
+installation:
+  location:
+    macos: ~/.tmux/plugins/
+    linux: ~/.tmux/plugins/ 
+    windows: ~/.tmux/plugins/ # requires Cygwin
+  flavours:
+    all:
+      default:
+        - ./*.tmuxtheme
+    latte:
+      default:
+        - ./catppucin-latte.tmuxtheme
+    frappe:
+      default:
+        - ./catppuccin-frappe.tmuxtheme
+    macchiato:
+      default:
+        - ./catppuccin-macchiato.tmuxtheme
+    mocha:
+      default:
+        - ./catppuccin-mocha.tmuxtheme
+  to: tmux
```

Comment:
It's a folder inside the location you're installing to. If you want to install to `~/.tmux/plugins/`, leave it as `.`. Currently this would install to `~/.tmux/plugins/tmux`.